code/datums/weather/weather_types/shroud_storm.dm

```diff
@@ -0,0 +1,41 @@
+/datum/weather/shroud_storm
+	name = "shroudstorm"
+	desc = "Zap."
+
+	telegraph_message = "<span class='danger'>You hear thunder in the distance, static electricity rising in the air, wind starts to pickup..</span>"
```

Comment:
Likewise for all of these really, even if you're copy-pasting this code, it should be updated for consistency